### PR TITLE
new packages: nnn, deadline, xdg-utils

### PR DESCRIPTION
**Pull request by JamsNJellies**
## Description of package
Add nnn and it's dependencies that are not allready in the repos
## Checklist

- [X] Latest upstream version.
- [X] I agree to maintain this package (**required**).
- [X] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/nnn/build [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
```
Comment (dylanaraps) [inline community/nnn/build]: Add a newline here.

File: community/nnn/build [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+make
+make strip install
```
Comment (dylanaraps) [inline community/nnn/build]: The package manager runs `strip` automatically remove this.

File: community/nnn/depends [hunk 3]
```diff
@@ -0,0 +1,3 @@
+file
+xdg-utils
```
Comment (dylanaraps) [inline community/nnn/depends]: Is this a mandatory dependency or is it recommended?

**Comment by dylanaraps** [comment]
Please split this into multiple pull requests (*one per package*). :+1:

**Comment by JamsNJellies** [comment]
Deadline in the title should be readline.
Xdg-utils is required for xdg-open.
That and file seem to be mandatory.
I think it can be built without readline
I'll split it into multiple pull requests when I have the time.

**Comment by dylanaraps** [comment]
`xdg-open` is optional as `NNN_OPENER` can be set to anything. `xdg-open`/`xdg-utils` are just the default.

**Pull request closed without merging**